xml/rmt_rmt-cli.xml

```diff
@@ -180,33 +179,33 @@
     <term><command>rmt-cli repos clean</command></term>
     <listitem>
      <para>
-      This command removes locally mirrored files of repositories which are not marked to be mirrored.
+      This command removes locally mirrored files of repositories that are not
+      marked to be mirrored.
      </para>
     </listitem>
    </varlistentry>
   </variablelist>
  </sect2>
 
  <sect2 xml:id="sec-rmt-tools-rmt-cli-repos-custom">
-  <title><command>repos custom</command></title>
+  <title><command>repos&nbsp;custom</command></title>
```

Comment:
This is a weird change. And if anyone copy-pastas this out of an HTML document into their terminal, they'll be unhappy with the result.

Reply:
The proofreader complained about a line break at this point in the TOC. I was attempting to fix it. It did not work; DAPS line-breaks it there anyway. 

I have reverted it.